XRTK-Core/ProjectSettings/ProjectSettings.asset

```diff
@@ -688,7 +690,7 @@ PlayerSettings:
   m_RenderingPath: 1
   m_MobileRenderingPath: 1
   metroPackageName: XRTK-Core
-  metroPackageVersion: 0.2.0.0
+  metroPackageVersion: 2019.3.11.0
```

Comment:
let's revert this

Reply:
Huh? What and how happened here. Reverting.